src/pydap/handlers/dap.py

```diff
@@ -46,12 +49,12 @@ def __init__(self, url):
         ddsurl = urlunsplit((scheme, netloc, path + '.dds', query, fragment))
         r = requests.get(ddsurl)
         r.raise_for_status()
-        dds = r.text.encode('utf-8')
+        dds = r.text.decode('ascii')
```

Comment:
Yeah, I think you're right. This is one of those cases that I discussed above where the code proper and the testing code don't use the same HTTP frameworks. So... the tests pass, but it's not really testing the right thing. Looking into a better solution to this...